common/src/main/java/hardcorequesting/common/quests/task/CompleteQuestTask.java

```diff
@@ -11,6 +11,8 @@
 import hardcorequesting.common.quests.task.client.TaskGraphic;
 import hardcorequesting.common.team.Team;
 import hardcorequesting.common.util.EditType;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
```

Comment:
Is this save to use in a forge environment?

Reply:
Yeah. There should be a part of the architectury plugin that transforms them into the forge equivalents when building the forge version. This is also something that has already been widely used in the common sub-project.